modules/core/src/matrix_wrap.cpp

```diff
@@ -1119,6 +1125,8 @@ size_t _InputArray::step(int i) const
     if (k == STD_VECTOR_CUDA_GPU_MAT)
     {
         const std::vector<cuda::GpuMat>& vv = *(const std::vector<cuda::GpuMat>*)obj;
+        if( i < 0 )
+            return 1;
```

Comment:
Is that correct? Why not assertion?

Reply:
It is used at several similar places, like in conditional that handles `STD_ARRAY_MAT`. As `step` (and `offset`) are declared with `-1` to be a default value of `i`, my guess, based on [this](https://github.com/opencv/opencv/blob/3.4.10/modules/core/src/matmul.dispatch.cpp#L83), is that `if ( i < 0 ) return 1;` is preferred over assertion to prevent `mat.step()` and `mat.offset()` calls from failing.

Reply:
maybe, it should be `sizeof(cuda::GpuMat>)`?